app/models/profile.rb

```diff
@@ -173,6 +173,15 @@ def in_person_verification_pending?
     in_person_verification_pending_at.present?
   end
 
+  def deactivate_due_to_gpo_expiration
+    raise 'Profile is not pending GPO verification' if gpo_verification_pending_at.nil?
+    update!(
+      active: false,
+      gpo_verification_pending_at: nil,
+      gpo_verification_expired_at: Time.zone.now,
```

Comment:
I was wondering if we want the field to be `expired_at` and let it apply to in person proofing as well, in which case we'd want to leave the `gpo_verification_pending_at` value in place to know what kind of proofing they were attempting. I suppose that could get complicated if they were doing in person *and* by mail.

Even if we leave it as `gpo_verification_expired_at`, I wonder if we want to leave the `gpo_verification_pending_at` value as a data trail.

Reply:
My concern with leaving `gpo_verification_pending_at` is we have a lot of other code that uses that to mean "this is an active GPO profile". Nulling it hides it from them.

Reply:
That makes sense. I would log the value in the analytics event then.

Reply:
Added `gpo_verification_pending_at` to analytics event in 6e49b411aadc0155b6121c51a8f065d7aede709e